images/makeproject/project/onechunk_workgen.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+IFS=' '
+filename='md5hashes.txt'
```

Comment:
Where is that file?

Reply:
Added it and changed the file name as requested.